Project-II-Calculator/calculator/src/less/index.less

```diff
@@ -0,0 +1,5 @@
+@import "variables.less";
```

Comment:
love seeing you use your LESS skills in this since you used your vanilla CSS skills in the previous part of this assignment. It's super important to keep finding ways to flex and use your previous skill sets. Well done